### PR TITLE
[#1006] Upgraded LCLS version to 2.2

### LINKED ISSUE #1006: Support the latest release of LCLS (Q4-2024)
Test with LCLS snapshot drivers when available: https://github.com/OpenLiberty/liberty-language-server

Update to final release version once integration is successful.

Check for any regression in LCLS behaviour.
Determine new LCLS behaviour to test and verify it behaves as expected (i.e. support for versionless features).

If any problems are discovered, open issues and track them in this issue.

- [x] https://github.com/OpenLiberty/liberty-tools-intellij/issues/967
- [x] https://github.com/OpenLiberty/liberty-tools-intellij/issues/1034
- [x] https://github.com/OpenLiberty/liberty-tools-intellij/issues/1035
- [x] #1017
- [x] Update LCLS dependencies to release version `2.2`

### PR DESCRIPTION
**Pull request by dessina-devasia**
Fixes #1006
Upgraded LCLS version to 2.2

### CODE REVIEW

**Review by turkeylurkey — DISMISSED** [review]

**Review by TrevCraw — APPROVED** [review]

**Review by TrevCraw — CHANGES_REQUESTED** [review]
Now that LCLS has been released, let's go ahead and bump to the release version `2.2`

File: build.gradle [hunk 1]
```diff
@@ -231,10 +231,10 @@ patchPluginXml {
         <li> When the contents of the Liberty tool window are too large to fit in its window, a scroll bar appears.
         <li> Helpful advice is displayed when no project appears in the Liberty tool window.
         <li> Added the Language Servers tool window, which you can use when you need technical support.
-        <li> Updated the <a href="https://github.com/OpenLiberty/liberty-language-server">Liberty Config Language Server</a> version to 2.1.2. For more information about the changes for version 2.1.2, refer to the following release notes:
+        <li> Updated the <a href="https://github.com/OpenLiberty/liberty-language-server">Liberty Config Language Server</a> version to 2.2. For more information about the changes for version 2.2, refer to the following release notes:
```
Comment (turkeylurkey) [inline build.gradle:234]: You can't change this here. These are the release notes for 24.0.3. Ask Trevor how to handle release notes for 24.0.12 (or whatever).

**Review by turkeylurkey — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (dessina-devasia) [reply build.gradle]: Sure @turkeylurkey. Reverted the modifications in change notes

**Review by turkeylurkey — APPROVED** [review]

**Comment by turkeylurkey** [comment]
I like `git rebase -i main` because it brings up an editor and you can squash commits etc.

**Comment by dessina-devasia** [comment]
Opened a similar PR https://github.com/OpenLiberty/liberty-tools-intellij/pull/1055

**Pull request closed without merging**